### ISSUE TITLE
"Anmeldung mit Tastendruck" ist aktiviert, daher kann JFritz nicht mit der Box kommunizieren.

### ISSUE THREAD

**Issue by torantal** [jfritz-org/jfritz#21]
Seit geschätzt v0.7.8 bricht jfritz den Start ab mit der Meldung ""Anmeldung mit Tastendruck" ist aktiviert, daher kann JFritz nicht mit der Box kommunizieren."

Meine Fritzbox 7390 mit FRITZ!OS 06.04 hat diese Funktion noch nicht einmal.

Neu installieren oder irgendein anderer Tipp?

![image](https://user-images.githubusercontent.com/6267029/75104059-a1a4a000-5604-11ea-8918-3b027660e123.png)
![image](https://user-images.githubusercontent.com/6267029/75104092-0bbd4500-5605-11ea-9c54-3a0f1c5d7c4b.png)

**Comment by robotniko** [comment]
Hi,

die neueste Firmware-Version für die 7390 ist die 06.86. 

Kommt der Fehler auch mit dieser Firmware-Version?

Grüße
Robert

**Comment by torantal** [comment]
Kann ich nicht sagen, da ich die Firmware nicht update, um meine LCR Funktionalität nicht zu verlieren.

In jedem Fall macht die Fehlermeldung bei meinem Modell ohne die Funktion "Anmeldung mit Tastendruck" keinen Sinn.

**Comment by robotniko** [comment]
Hi, 

könntest du bitte einige Tests bei dir durchführen?
- ZIP entpacken
- Im entpackten Verzeichnis folgende Befehle ausführen und mir jeweils den Output an support@jfritz.org zusenden:
  - java -jar FBTR064.jar fritz.box
  - java -jar FBTR064.jar fritz.box 'deinPasswort'
  - In der FritzBox mal auf Benutzername und Passwort umstellen und dann 
    - java -jar FBTR064.jar fritz.box 'deinPasswort' 'deinUser'

(die Hochkommas werden benötigt, falls dein Passwort Sonderzeichen enthält)

[FBTR064.zip](https://github.com/jfritz-org/jfritz/files/4301771/FBTR064.zip)

**Comment by torantal** [comment]
Kann ich im Prinzip schon machen, eine kurze Erklärung, was da im .jar drin ist und was es macht wäre vorher nett. Ein 20 MB File mit knapp 12000 Dateien in knapp 1200 Ordnern kann ja sonstwas anstellen....

**Comment by robotniko** [comment]
Klar, es macht exakt dies: https://github.com/jfritz-org/FritzTR064/blob/feature/TestLoginMethod/src/main/java/de/bausdorf/avm/tr064/TestLoginMethod.java

Kurz zusammengefasst: Es spricht die TR064-API-Endpunkte deiner FritzBox an, um herauszufinden ob:
- Anonymer login möglich ist
- Login mit Button aktiviert ist
- Aktuelle Benutzerrechte
- Aktuell eingeloggter Benutzer

**Comment by torantal** [comment]
Hier geht auch, oder?

```
java -jar FBTR064.jar fritz.box
14:37:27.833 - Connecting to FritzBox at fritz.box
14:37:29.747 - Detecting login method
14:37:29.750 - Cannot detect login method. Service 'LANConfigSecurity:1' not available. Returning LoginMethod.UNKNOWN


java -jar FBTR064.jar fritz.box XXXXXXXX
14:38:32.178 - Connecting to FritzBox at fritz.box
14:38:34.060 - Detecting login method
14:38:34.060 - Executing action: X_AVM-DE_GetAnonymousLogin
14:38:34.519 - NewX_AVM-DE_AnonymousLoginEnabled: 1
14:38:34.520 - Anonymous login enabled: true
14:38:34.520 - Field 'NewX_AVM-DE_ButtonLoginEnabled' of action 'X_AVM-DE_GetAnonymousLogin' not available.
14:38:34.520 -
14:38:34.520 - Executing action: X_AVM-DE_GetCurrentUser
14:38:34.920 - NewX_AVM-DE_CurrentUsername:
14:38:34.921 - Field 'NewX_AVM-DE_CurrentUserRights' of action 'X_AVM-DE_GetCurrentUser' not available.


java -jar FBTR064.jar fritz.box XXXXXXXX admin
14:42:04.688 - Connecting to FritzBox at fritz.box
14:42:06.272 - Detecting login method
14:42:06.272 - Executing action: X_AVM-DE_GetAnonymousLogin
14:42:06.373 - NewX_AVM-DE_AnonymousLoginEnabled: 0
14:42:06.374 - Anonymous login enabled: false
14:42:06.377 - Field 'NewX_AVM-DE_ButtonLoginEnabled' of action 'X_AVM-DE_GetAnonymousLogin' not available.
14:42:06.378 -
14:42:06.379 - Executing action: X_AVM-DE_GetCurrentUser
14:42:06.770 - NewX_AVM-DE_CurrentUsername: admin
14:42:06.770 - Field 'NewX_AVM-DE_CurrentUserRights' of action 'X_AVM-DE_GetCurrentUser' not available.
```

**Comment by robotniko** [comment]
Jap, vielen Dank.
